### PR TITLE
Add query for multiple tables

### PR DESCRIPTION
**Pull request by abdozayan12**
Screenshot 2023-08-17 124928
Screenshot 2023-08-17 125001
Screenshot 2023-08-17 124534
Screenshot 2023-08-17 124545
Screenshot 2023-08-17 124607
Screenshot 2023-08-17 124616
Screenshot 2023-08-17 124628
Screenshot 2023-08-17 124645
Screenshot 2023-08-17 135118

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -49,4 +54,40 @@ SELECT neutered, MAX(escape_attempts) AS max_escape_attempts FROM animals GROUP
...
+SELECT animals.name AS digimon_name
+FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
+SELECT animals.name AS animal_name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
```
Comment (V-Blaze) [inline queries.sql:86]: - [ ] You have done great so far :+1: :+1: But currently, you are missing the result screenshot for this query `List all animals owned by Dean Winchester that haven't tried to escape.` and as part of the requirement, you are meant to include all the query result screenshot. I'll suggest you update this pull request with the missing screenshot. :small_airplane: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @abdozayan12 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Correct data inserted into the database✔✔
- Queries are written correctly :heavy_check_mark: 
- Screenshots show correct results :heavy_check_mark: 
- Readme looks professional :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @abdozayan12, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Repo looks professional 

 ## Required Changes ♻️
_Check the comments under the review._
- Please make the requested change by the previous reviewer before resubmitting the PR.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdozayan12** [comment]
@afanimartin 
I have added the missing screenshot
Screenshot 2023-08-17 135118

**Comment by bellom** [comment]
@abdozayan12 - 

### Ongoing SO.....

Kindly wait for our review - thank you.

**Comment by bellom** [comment]
Hello @abdozayan12 👋,

_This is an additional review as a result of the second opinion request._

Sorry for the inconvenience 🙏 , yes, you added it an hour ago 👍.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few mins!

Keep rocking 🚀!

**Review by afanimartin — APPROVED** [review]
Hi @abdozayan12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Screenshot added as requested 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**